### PR TITLE
feat(parser): add support for API Gateway HTTP API #434

### PR DESCRIPTION
**Pull request by risenberg-cyberark**
Support for API GW v2 proxy schema & envelope.
see issue #434

### CODE REVIEW

**Comment by xpayn** [comment]
Good job! :)

File: aws_lambda_powertools/utilities/parser/models/apigwv2.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+class RequestContextV2(BaseModel):
+    accountId: str
+    apiId: str
+    authorizer: Optional[RequestContextV2Authorizer]
+    domainName: str
+    domainPrefix: str
+    requestId: str
+    routeKey: str
+    stage: str
+    property
```
Comment (heitorlessa) [inline aws_lambda_powertools/utilities/parser/models/apigwv2.py:54]: This seems off - perhaps a typo?

```suggestion

```

**Review by heitorlessa — CHANGES_REQUESTED** [review]
One tiny change that it seems to be a typo on a property that doesn't exist.

**Review by heitorlessa — APPROVED** [review]

**Comment by heitorlessa** [comment]
merging - thanks a lot @risenberg-cyberark ;)

**Pull request merged**